projects/vdk-plugins/airflow-provider-vdk/README.md

```diff
@@ -1,3 +1,62 @@
-## Versatile Data Kit Airflow provider
+# Versatile Data Kit Airflow provider
 
 A set of Airflow operators, sensors and a connection hook intended to help schedule Versatile Data Kit jobs using Apache Airflow.
+
+# Usage
+
+To install it simply run:
+```
+pip install airflow-provider-vdk
+```
+
+Then you can create workflow of data jobs like this:
```

Comment:
Maybe add a note that the jobs need to be deployed before running the DAG, in case users miss this part in the linked example.

Reply:
```suggestion
Then you can create a workflow of data jobs (deployed by VDK Control Service) like this:
```